auto_submit/test/requests/check_pull_request_test.dart

```diff
@@ -3,24 +3,26 @@
 // found in the LICENSE file.
 
 import 'package:auto_submit/requests/check_pull_request.dart';
+import 'package:github/github.dart';
 import 'package:shelf/shelf.dart';
 import 'package:test/test.dart';
 
 import './github_webhook_test_data.dart';
 import '../requests/github_webhook_test_data.dart';
+import '../src/request_handling/fake_pubsub.dart';
 import '../src/service/fake_config.dart';
 import '../src/service/fake_github_service.dart';
 
 void main() {
   group('Check CheckPullRequest', () {
```

Comment:
This PR is missing tests.

1. You can update your existing tests validating merges, and ensure the topic is empty
2. For tests without merges, you can ensure the topic still has the message
3. For tests removing the label, you should validate the message was removed

Reply:
Thank you Casey for listing ideas of how to update the tests! I updated the unit tests accordingly. For each unit test, i also created two pull requests to test the for logic we newly added to make check_pull_request handler parallelizable.